### ISSUE TITLE
Will react-bootstrap support Bootstrap 4 in near future?

### ISSUE THREAD

**Issue by changLiuUNSW** [react-bootstrap/react-bootstrap#2861]
We really like this library especially the awesome accessibility support and we want to use it in our new Bootstrap 4 project. 
We have reviewed another React library for Bootstrap 4 but we cannot use it because the accessibility does not meet accessibility requirement of our company (Bank).
We know react-bootstrap is highly customizable and some components may be compatible with Bootstrap 4. However we still hope react-bootstrap can officially support Bootstrap 4 so that we can easily plugin and play.
Does the react-bootstrap team have the plan to support Bootstrap 4 in near future?

**Comment by jquense** [comment]
we are working on it long term. We are a bit limited in personpower though so its a bit slow

**Comment by taion** [comment]
See https://github.com/react-bootstrap/react-bootstrap/issues/1187, https://github.com/react-bootstrap/react-bootstrap/issues/2829, and https://github.com/react-bootstrap/react-bootstrap/pull/2752.

**Comment by sw-yx** [comment]
would be happy to help.
